### PR TITLE
Repro for empty scene issues

### PR DESCRIPTION
**Pull request by sritchie**
Okay, I've got these reproducing WITHOUT any of the live-reload shenanigans. Both cases are mysterious. See the inline comments for an explanation.

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -40,12 +33,18 @@ function App() {
         show={show}
         setShow={setShow}
       />
+      {/* <div ref={setContainer} key={width.toString()} className="h-100"> */}
```
Comment (sritchie) [inline example/src/App.tsx:36]: If you

1. undo the change below
2. uncomment this line
3. comment out the line directly below

to set up a situation where changing the width changes the key, then you'll see the "Cannot read properties of undefined" on any width change.

File: example/src/App.tsx [hunk 2]
```diff
@@ -40,12 +33,18 @@ function App() {
...
       <div ref={setContainer} className="h-100">
         {show && container && (
           <MB.Mathbox
             container={container}
             ref={setup}
-            options={mathboxOptions}
+            options={{
```
[the remark this answers is not in the document]
Comment (sritchie) [reply example/src/App.tsx:42]: Played a bit more and it turns out you don't need to "inline" the `mathboxOptions`; simply moving the `const mathboxOptions = ...` inside of `App`, ie,

```
function App() {
...
const mathboxOptions = ...
...
}
```

will cause the same behavior.

**Comment by ChristopherChudzicki** [comment]
Going to close this.. should be resolved by some combo of #20 and #27
